pkg/loop/internal/pb/capabilities_registry.proto

```diff
@@ -22,6 +23,22 @@ enum ExecuteAPIType {
   EXECUTE_API_TYPE_CALLBACK = 2;
 }
 
+
+// DON has arguments for [github.com/smartcontractkit/chainlink-common/pkg/capabilities.DON].
+message DON {
+  string id = 1;
+  repeated bytes members = 2; // [membern][32]byte
+  uint32 f = 3; // uint8
+  bytes config  = 4;
+}
+
+// GetLocalNode has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.CapabilitiesRegistry.GetLocalNode].
+message GetLocalNodeReply {
+  bytes peerID = 1; // [32]byte
+  DON workflowDON = 2;
```

Comment:
it's worth a comment here that a node is asymmetric in membership: it only allowed to be a member of 1 workflow don but any number of capabilities dons, and IIUC that is enforced by the contrace